recipes/isitgr/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "isitgr" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mishakb/ISiTGR/archive/{{ version }}.tar.gz
+  sha256: ebf53b46ce3682a0154f3e713e8d320f5f7588fe1cb7a6fe5c9db8d1cad78b59
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('fortran') }}  # [not win]
+#    - m2w64-toolchain  # [win]
+    - git  # [not win]
+#    - m2-git  # [win]
+    - make  # [not win]
+#    - m2-make  # [win]
+    - python
+    - setuptools
+    - pip
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - numpy
+    - six  # TODO: remove for next update
+    - mock
+    - setuptools  # for an import of pkg_resources
+    - sympy >=1.0
+    - scipy >=1.0
+    - {{ compiler('fortran') }}  # [not win]
+
+test:
+  imports:
+    - isitgr
```

As it's simple, I would suggest removing `run_test.sh`/`run_test.bat ` and putting:
```suggestion
    - isitgr
  commands:
    - python {{ RECIPE_DIR }}/test_isitgr.py
```